src/UnitTests/test_quote_prov.cpp

```diff
@@ -111,11 +146,12 @@ static void GetCertsTest()
         0x05, 0x05, 0x02, 0x04, 0x01, 0x80, 0x01, 0x00,
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
     };
+

```

Comment:
This comment is not specific to Windows test case. It applies to both linux and Windows.

This test seems to assume the specific qe_id to have specific CPUSVN values, which are not fixed values. This test will fail when the CPUSVN is updated.

Reply:
As long as we don't remove this entry from the production PCK Caching Service, this should still work right?

Reply:
is that qe_id for a real machine? Or a dummy one?
If it's a dummy, it should be ok as long as you keep it in your table.

Reply:
Its a dummy one I believe.